code/modules/mining/equipment/wormhole_jaunter.dm

```diff
@@ -56,27 +64,24 @@
 	if(. & EMP_PROTECT_SELF)
 		return
 
-	var/mob/M = loc
-	if(istype(M))
-		var/triggered = FALSE
-		if(M.get_item_by_slot(ITEM_SLOT_BELT) == src)
-			if(power == 1)
-				triggered = TRUE
-			else if(power == 2 && prob(50))
-				triggered = TRUE
+	var/triggered = FALSE
+	if(power == 1)
+		triggered = TRUE
+	else if(power == 2 && prob(50))
+		triggered = TRUE
 
-		if(triggered)
-			M.visible_message(span_warning("[src] overloads and activates!"))
-			SSblackbox.record_feedback("tally", "jaunter", 1, "EMP") // EMP accidental activation
-			activate(M)
+	var/mob/M = loc
+	if(istype(M) && triggered)
+		M.visible_message(span_warning("[src] overloads and activates!"))
+		SSblackbox.record_feedback("tally", "jaunter", 1, "EMP") // EMP accidental activation
+		activate(M, FALSE, TRUE)
+	else if(triggered)
+		visible_message(span_warning("The [src.name] overloads and activates!"))
+		activate()
 
 /obj/item/wormhole_jaunter/proc/chasm_react(mob/user)
-	if(user.get_item_by_slot(ITEM_SLOT_BELT) == src)
-		to_chat(user, span_notice("Your [name] activates, saving you from the chasm!"))
-		SSblackbox.record_feedback("tally", "jaunter", 1, "Chasm") // chasm automatic activation
-		activate(user, FALSE)
-	else
-		to_chat(user, span_userdanger("[src] is not attached to your belt, preventing it from saving you from the chasm. RIP."))
```

Comment:
Ditto

Reply:
Most of this was moved to the chasm.dm  The line about not being attached to your belt was never used because the chasm would check if the jaunter was in the belt slot and if it wasn't then the mob would fall without ever triggering the `to_chat` proc.  This is because the chasm never checked if the jaunter was located in a different inventory slot.